pkg/webui/lib/shared-messages.js

```diff
@@ -106,12 +106,19 @@ export default defineMessages({
   frequencyPlan: 'Frequency Plan',
   fNwkSIntKey: 'FNwkSIntKey',
   nwkSKey: 'NwkSKey',
+  gatewayAutoUpdate: 'Automatic Updates',
```

Comment:
```suggestion
  automaticUpdates: 'Automatic Updates',
```

similar changes for the rest of messages